src/Core/src/Platform/Android/MauiWebView.cs

```diff
@@ -15,6 +17,24 @@ public MauiWebView(WebViewHandler handler, Context context) : base(context)
 			_handler = handler ?? throw new ArgumentNullException("handler");
 		}
 
+		public override bool OnTouchEvent(MotionEvent? e)
+		{
+			RequestDisallowInterceptParentTouchEvent();
+
+			return base.OnTouchEvent(e);
+		}
+
+		void RequestDisallowInterceptParentTouchEvent()
+		{
+			if (Parent is null)
+				return;
+
+			var viewPager2 = Parent.GetParentOfType<ViewPager2>();
+
+			if (viewPager2 is not null)
+				Parent?.RequestDisallowInterceptTouchEvent(HorizontalScrollBarEnabled);
```

Comment:
We already know that Parent is not null, correct? No need to the ?
```suggestion
				Parent.RequestDisallowInterceptTouchEvent(HorizontalScrollBarEnabled);
```